scripts/node_pair_extraction_helpers.py

```diff
@@ -3,6 +3,62 @@
 from file_helpers import *
 from ortholog_helpers import *
 
+def add_valid_edges(list_of_species_nodes, lines):
```

Comment:
valid is too vague, either rename or define "valid" on top of the function in a comment